MigrationTool/Program.cs

```diff
@@ -0,0 +1,277 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using Rampastring.Tools;
+
+namespace MigrationTool;
+
+internal sealed class Program
+{
+    private enum Version
+    {
+        Begin,
+        V_2_11_0,
+        V_2_11_1,
+        V_2_11_2,
+        V_2_12_0,
+        V_2_12_1,
+        V_2_12_5,
+        Latest,
+        End
+    }
```

Comment:
using enum for this looks like an overcomplication without actual use to me

Reply:
I don't think so because it allows step-by-step migration from one version to another and to the latest in the end.

Reply:
this is not inherent to enums only, it's one of the ways to do that. instead just make each version migration a function and call it

Reply:
My intention when telling it to mah_boi was to mark the actually resolved feedback, i.e. when all participants are obviously agreeing. Like for small corrections that don't require any further input and were addressed, not for actually debatable changes and resolutions. In those cases obviously one shouldn't resolve the conversation and better leave that to maintainers.